eeg_flow/utils/bids.py

```diff
@@ -92,4 +92,4 @@ def get_xdf_folder(
     root = ensure_path(root, must_exist=True)
     check_type(participant, (str,), "participant")
     check_type(group, (str,), "group")
-    return root / f"sub-{participant}-{group}"
+    return root / f"sub-{participant}-{group}" / "eeg"
```

Comment:
So now you have 2 roots, either passive or active? Correct?

Reply:
yes! which would be set from step0 by the user

Reply:
OK, that's good then. Why the extra "eeg" here?

Reply:
we're starting to use the behavioral files as well (videos and olstat), so I was thinking of placing them in the raw subfolder of each participant too

>sub-P01-G1
>>eeg
>>beh